### DISCUSSION TITLE
pdf2word

### DISCUSSION THREAD

**Discussion by yiyahei-eng** [PaddlePaddle/PaddleOCR#14426] [Q&A]
下载代码运行后，非矢量化的pdf转成word就是图片，是正常的吗？

**Comment by yiyahei-eng** [comment]
@GreatV 我想要将非矢量化的pdf转换成为可编辑的word，本来我是使用paddle ocr提取文字，然后PP-StructureV2 识别版面分析，最后输出原本的排版，但是输出的word是空白的

**Reply by GreatV** [reply]
正常，版面分析的时候错误地把文字段落识别成了图片。
